drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -174,6 +220,13 @@ protected void loadContent() {
             baseView.hideBusyIndicator();
             view.setContent(model.getHeadersMap(), model.getRowsMap());
             createOriginalHash(model.hashCode());
+//            if (PlaceStatus.CLOSE.equals(placeManager.getStatus(RightPanelPresenter.IDENTIFIER))) {
+//                placeManager.goTo(RightPanelPresenter.IDENTIFIER);
+//            }
```

Comment:
Commented code can be removed.

Reply:
Fixed with
[DROOLS-2780] Refactored RightPanelView/RightPanel interaction. Removing access modifier for testing purpose. Renaming mocked instances.